_posts/2024-03-06-happy-birthday.md

```diff
@@ -0,0 +1,43 @@
+"Hello, you probably don't know me, but I've been working on a project to clone TTD for a while." he said, more or less.
+He sent me a copy of what he'd been working on, and it was very exciting indeed.
+It was a fully functional version of Transport Tycoon Deluxe, written in C.
+Admittedly incomplete at that time, it was still quite remarkable.
+Ludde didn't want to release this to the public yet, and wasn't entirely sure what he was going to do with it.
+(He'd been working on it since 30th June 2002, so I guess that is technically the birthday of what would become OpenTTD...)
+
+<!-- more -->
+
+Ludde made more progress with the project over the coming year, and it looks like we even attempted some multiplayer games (not too reliable, especially over my dial-up connection at the time).
+Eventually, when he was happy with what he had created, he agreed to allow me to release the game as open source.
+Coincidentally, this happened exactly a year after I'd first spoken to him, on the 6th March 2004.
+But first, it needed a name.
+I don't think I thought very much about it, but decided that "OpenTTD" had a nice ring to it, so registered a SourceForge project, created an OpenTTD forum on TT-Forums, and [set it loose](https://www.tt-forums.net/viewtopic.php?f=29&t=6559).
+OpenTTD 0.1 was released.
+You can [still download it](https://cdn.openttd.org/openttd-releases/0.1.0/) if you want to see what it was like, though it's not particularly straightforward to run!
+
+Things really got going after this, and a community started to form with enthusiastic developers fixing bugs, adding in new features, and smoothing off the rough edges.
+Ludde was, I think, a bit taken aback by how popular it proved, and even rejoined the development effort for a while.
+A read through the [old changelogs](https://wiki.openttd.org/en/Archive/OpenTTD/) reveals just how many features were added over a very short period of time.
+Quick wins like higher vehicle limits came in very quickly, and support for TTDPatch's NewGRF format started to be functional just four months later.
+Large maps, improved multiplayer, better pathfinders, improved TTDPatch compatibility, and of course, ports to a great many different operating systems, such as Mac OS X, BeOS, MorphOS and OS/2.
+It was a very exciting time to be a TTD fan!
+
+Within six years, ambitious projects to create free replacements for the original TTD graphics, sounds and music sets were complete, and OpenTTD finally had its 1.0 release.
+And while we may not have the same frantic addition of new features we had in 2004, there have still been massive improvements to the code, with plenty of exciting new features over the years, with major releases every year since 2008.
+The move to GitHub in 2018 and the release of OpenTTD on Steam in 2021 have also re-energised development efforts, with thousands of people now enjoying playing the game regularly.
```

Comment:
Maybe a reference to that 14.0 shows how active OpenTTD is these days, with more features in 1 version than there has been in years? Not sure how to word it.